### PR TITLE
Update report-false-positives-and-false-negatives.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a section on how to turn off the built in experience

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6956

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6955

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6957

**Comment by msbemba** [comment]
All three issues are covered by this PR

File: microsoft-365/security/office-365-security/report-false-positives-and-false-negatives.md [hunk 1]
```diff
@@ -46,6 +43,19 @@ In Microsoft 365 organizations with mailboxes in Exchange Online or on-premises
 
 - For more information about reporting messages to Microsoft, see [Report messages and files to Microsoft](report-junk-email-messages-to-microsoft.md).
 
+### Turn off the built-in reporting experience
+
+We don't recommend the built-in reporting experience in Outlook because it can't use the [user submission policy](./user-submission.md). We recommend using the Report Message add-in or the Report Phishing add-in instead.
+
+You need to be assigned permissions before you can run this cmdlet.  To find the permissions required to run any cmdlet or parameter in your organization, see [Find the permissions required to run any Exchange cmdlet](/powershell/exchange/find-exchange-cmdlet-permissions).
+
+Run the below powershell command to disable the built-in reporting experience in Outlook on the web.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/report-false-positives-and-false-negatives.md]:
```suggestion
You need to be assigned permissions before you can run this cmdlet. To find the permissions required to run any cmdlet or parameter in your organization, see [Find the permissions required to run any Exchange cmdlet](/powershell/exchange/find-exchange-cmdlet-permissions).

Run the following PowerShell command to disable the built-in reporting experience in Outlook on the web:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**